modules/nf-core/parabricks/applybqsr/main.nf

```diff
@@ -15,36 +14,36 @@ process PARABRICKS_APPLYBQSR {
     output:
     tuple val(meta), path("*.bam"), emit: bam
     tuple val(meta), path("*.bai"), emit: bai
-    path "versions.yml"           , emit: versions
+    path "versions.yml", emit: versions
```

Comment:
That's not harshil aligned anymore

Reply:
https://github.com/nextflow-io/language-server/issues/71